### DISCUSSION TITLE
how specify colours for links on epub and other formats also

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#12441] [Q&A]
* i have bunch of epubs
* they contain tappable/clickable links [ i mean tapping text will provide prompt to copy, open, qr and cancel of external link ]
* how do i specify link text or link text background colour?

**Comment by Frenzie** [comment]
There are a few presets in the style tweaks menu, but to give them a background color I think you'd have to write a custom one.

**Reply by ghost** [reply]
@Frenzie thanks for the tip and prompt reply ;)
* i am unaware of koreader terminology 
* underline, bold, italic, .... options for **links style tweaks** are more than enough for me
